README.md

```diff
@@ -1196,6 +1196,39 @@ peered with.
 
 `peer_vpc` defines the identifier or name of the remote VPC.
 
+### Datasource Service Component
+```hcl-terraform
+data "aiven_service_component" "sc1" {
+    project = aiven_kafka.project1.project
+    service_name = aiven_kafka.service1.service_name
+    component = "kafka"
+    route = "dynamic"
+}
+```
+
+`project` and `service_name` define the project and service the service component
+belongs to.
+
+`component` is a service component name.
+
+`route` is network access route.
```

Comment:
Forgot to comment earlier but this should list different route options and what those actually mean. Something like

Route may be one of `dynamic`, `public`, and `private`. Usually you'll want to use `dynamic`, which for services that are not in private network identifies the regular public DNS name of the service and for services in private network the private DNS name. If the service is in private network but has also public access enabled the `public` route type can be used to get the public DNS name of the service. The `private` option should typically not be used.